client/src/partials/cash/cashboxes/cashboxes.html

```diff
@@ -186,4 +186,4 @@
       </div>
     </div>
   </div>
-</main>
+</main>
```

Comment:
Could you configure your editor to always add the last newline character (`\n`)?  Having it strip the last newline leads to polluted diffs.  [Here's a discussion about it](http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file).